docs/deploy.md

```diff
@@ -178,3 +178,23 @@ frontend:
   ```sh
   docker run -itp 3000:3000 --name=docsify -v $(pwd):/docs docsify/demo
   ```
+
+## Kinsta Static Site Hosting
+You can deploy **Docsify** as a Static Site on [Kinsta](https://kinsta.com/static-site-hosting/).
+
+1. Login or create an account to view your [MyKinsta](https://my.kinsta.com/) dashboard.
+
+2. Authorize Kinsta with your Git provider.
+
+3. Select **Static Sites** from the left sidebar and press **Add sites**.
+
+4. Select the repository and branch you want to deploy.
+
+5. During the build settings, Kinsta will automatically try to fill out the **Build command**, **Node version**, and **Publish directory**. If it won't, fill out the following:
+
+   - Build command: leave empty
+   - Node version: `18.16.0`
```

Comment:
Specifying a node version in these docs means these instructions will become outdated as nobody is going to update `18.16.0` to the latest stable build. Will Kinsta not choose a reasonable node version by default? If so, consider changing `18.16.0` to something like `leave on default selection`.